packages/cms-cli/lib/prompts.js

```diff
@@ -58,10 +59,28 @@ const PORTAL_API_KEY = {
   },
 };
 
+const AUTH_METHOD = {
+  type: 'list',
+  name: 'authMethod',
+  message: 'Choose authentication method',
+  default: AUTH_METHODS.oauth,
+  choices: Object.keys(AUTH_METHODS).map(method => {
+    const authMethod = AUTH_METHODS[method];
+    return {
+      value: authMethod,
+      name: authMethod
+        .split(CAPITAL_LETTER_REGEX)
+        .join(' ')
+        .toLowerCase(),
+    };
+  }),
```

Comment:
Instead of generating the choices, I think this is a situation where it is better to just spell them out. Doing so makes it easy to change the label used, which I think should be adjusted to be "OAuth2" and "API Key."